### DISCUSSION TITLE
Is it possible to install pixi without admin rights on Windows?

### DISCUSSION THREAD

**Discussion by LSPedrosa** [prefix-dev/pixi#1598] [Q&A] [answered]
Is it possible to install pixi as user  on Windows? If yes, how?

I would like to install maybe using pipx and try pixi to replace poetry.

I have investigated a bit, but did not find anything conclusive.

**Comment by baszalmstra** [comment]
The regular installer from pixi.sh should not require admin privileges.

**Reply by LSPedrosa** [reply]
Ok, thanks @baszalmstra for the answer.
Maybe the problem is because I'm trying to install behind a network which is blocking the pixi.sh website.
I tried downloading the installation file "install.ps1" and running it locally but it didn't work.

**Reply by baszalmstra** [reply, marked as answer]
Pixi is just a single binary that you can get from the github releases page. If you download the executable and add it to your PATH  it should just work.
